operator/builtin/input/azure/event_hub.go

```diff
@@ -0,0 +1,106 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package azure
+
+import (
+	"context"
+	"fmt"
+	"sync"
+
+	azhub "github.com/Azure/azure-event-hubs-go/v3"
+	"go.uber.org/zap"
+)
+
+// Eventhub provides methods for reading events from Azure Event Hub.
+type EventHub struct {
+	AzureConfig
+	Persist *Persister
+	WG      sync.WaitGroup
+	Handler func(context.Context, *azhub.Event) error
```

Comment:
Let's not export unless necessary.

Reply:
These are called by the operators in two seperate packages, `eventhub` and `loganalytics`.